tests/test_correct_year.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+    test_correct_year
```

Comment:
Oh sorry. I had misread this tries to test gettext builder. Please forget my comment.